conftest.py

```diff
@@ -243,7 +243,6 @@ def fail_if_open_message_boxes_left():
         except RuntimeError:
             pass
 
-
```

Comment:
This change seems wrong to me. Top-level functions should be separated by 2 newlines according to pep8.

Reply:
ouch, but there are more

https://github.com/Open-MSS/MSS/issues/2093